CHANGES.md

```diff
@@ -43,6 +43,10 @@ To be released.
     [[#3144]]
  -  Added `IActionContext.GasLimit()` method.
     [[#3144]]
+ -  Added `ITransaction.MaxGasPrice` property.
+    [[#3156]]
+ -  Added `ITransaction.GasLimit` property.
+    [[#3156]]
```

Comment:
Basically, adding interface methods seems to the API breaking changes because its implementation should be updated.

Reply:
This is all nullable so do not directly affect implementation I guess. but should I mark `Breaking change` on those comments?